src/contrib/ding-dong/ding-dong.spec.ts

```diff
@@ -54,7 +54,7 @@ test('isMatchConfig {mention: false}', async t => {
 
     const mentionMessage = await new Promise<Message>(resolve => {
       room.once('message', resolve)
-      fixture.mocker.player.say('ding', [fixture.mocker.bot]).to(fixture.mocker.room)
+      fixture.mocker.player.say('ding', [ fixture.mocker.bot ]).to(fixture.mocker.room)
```

Comment:
Please submit another PR to fix the linting, instead of including the lining fixes in this PR, to make this PR only related to the MQTT plugin.

Reply:
Okay, I will do it later and close this pull.